NonsoAmadi10/Auto-Mart
# Ft post car ad #166229205

### What does this PR do?
- This PR is an implementation of the feature that allows a user create a car sale advert

### Description of Task Taken to Complete??
- Setup folder directories
- Setup test case
- Install mutler
- Install cloudinary
- Create an account on clodinary
- Configure mutler
- Configure cloudinary
- Implement feature
- Run test against feature
- Refractor code if test passes

### How should this be manually tested??
- Clone this repo
- Open a command line terminal
- Run ` npm dev-start` to start the server
- Open postman and ` POST: localhost:8000/api/v1/car`
- Pass in ` { photo, price, manufacturer, model, bodyType, state } in the request body to see the result
- Alternatively, ` run npm test ` to see code pass the test

### Any Background Context you want to provide??
- You must be a registered User and logged in to use this route. To do this see #25 and #28 
- On postman, the request body must be a form-data and not raw

### What are the relevant pivotal tracker stories??
[166229205](https://www.pivotaltracker.com/story/show/166229205)
